src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,228 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.exceptions.EmptyCommandException;
+import duke.exceptions.EmptyFindException;
+import duke.exceptions.InvalidCommandException;
+import duke.exceptions.MissingTimeException;
+import duke.exceptions.TaskCompletionException;
+import duke.exceptions.TaskDeletionException;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+/**
+ * Stores the current list of tasks.
+ */
+
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
+        return this.tasks;
+    }
+
+    /**
+     * Prints all the tasks line by line to the user.
+     * @return All tasks in order.
+     */
+    public String displayTasks() {
+        StringBuilder res = new StringBuilder("Here are the tasks in your list:");
```

Comment:
Perhaps, it may be beneficial to provide a full word (such as result) as the name of this variable instead of an abbreviation. Otherwise, the provision of a brief explanation of this variable's name could be useful to improve readability.

I have noticed this issue regarding abbreviation in variable naming in other parts of code too.